.github/workflows/release.yml

```diff
@@ -119,12 +184,6 @@ jobs:
         env:
           CIBW_ARCHS: ${{ matrix.arch }}
           CIBW_SKIP: pp* *musllinux* *-manylinux_i686
-          CIBW_TEST_SKIP: "cp38-macosx_arm64"
-          CIBW_ENVIRONMENT_LINUX:
-            GEOS_VERSION=${{ env.GEOS_VERSION }}
-            GEOS_INSTALL=/host${{ runner.temp }}/geos-${{ env.GEOS_VERSION }}
-            GEOS_CONFIG=/host${{ runner.temp }}/geos-${{ env.GEOS_VERSION }}/bin/geos-config
-            LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/host${{ runner.temp }}/geos-${{ env.GEOS_VERSION }}/lib
```

Comment:
Setting those env variables apparently don't seem to be needed in practice for linux?

Reply:
(that's maybe because in the docker image we are just installing GEOS in the expected system default location?)

Reply:
These are not needed in an image since it's installed to a "default" path  `/usr/local` which is automatically discovered.